files/fr/web/api/htmliframeelement/csp/index.html

```diff
@@ -0,0 +1,26 @@
+---
+title: HTMLIFrameElement.csp
+slug: Web/API/HTMLIFrameElement/csp
+browser-compat: api.HTMLIFrameElement.csp
+translation_of: 'HTMLIFrameElement.csp'
+---
+<p>{{SeeCompatTable}}{{APIRef("HTML DOM")}}</p>
+
+<p>La propriété <strong><code>csp</code></strong> de l'interface {{domxref("HTMLIFrameElement")}} spécifie la <a href="/fr/docs/Web/HTTP/CSP">Content Security Policy</a> (ou stratégie de sécurité du contenu) qu'un document embarqué doit accepter pour pouvoir s'auto-appliquer.</p>
+
+<h2 id="syntax">Syntaxe</h2>
+
+<pre class="brush: js">var <em>csp</em> = HTMLIFrameElement.csp
+HTMLIFrameElement.csp = <em>csp</em></pre>
+
+<h3 id="value">Valeur</h3>
+
+<p>Une content security policy (stratégie de sécurité du contenu).</p>
```

Comment:
I'd leave only the French expression here -(I'd guess we can expect a reader who has made it this far that they know what a CSP is).
In order to be slightly better than the English docs, I would also add that this one is a string. (https://w3c.github.io/webappsec-cspee/#dom-htmliframeelement-csp) (or else, I wouldn't know what to do with such a value).

Reply:
Agreed!
Fixed in https://github.com/mdn/translated-content/pull/1790/commits/f4d36cc635776c889b2ba9c37bd2e9743752a4f4